frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueCard.tsx

```diff
@@ -84,26 +121,30 @@ export const MissionQueueCard = ({ order, mission, onDeleteMission }: MissionQue
         <StyledMissionCard key={mission.id} style={{ boxShadow: tokens.elevation.raised }}>
             <HorizontalContent>
                 <HorizontalNonButtonContent onClick={routeChange}>
-                    <CircularCard style={{ background: fillColor }}>
+                    <CircularCard id="item1" style={{ background: fillColor }}>
                         <Typography variant="body_short_bold">{order}</Typography>
                     </CircularCard>
-
-                    {mission === placeholderMission ? (
-                        <PaddingLeft>
-                            <DotProgress size={48} color="primary" />
-                        </PaddingLeft>
-                    ) : (
-                        <Button variant="ghost" fullWidth>
-                            <Typography variant="body_short_bold">{mission.name}</Typography>
-                        </Button>
-                    )}
-                    <Typography variant="caption" color={tokens.colors.text.static_icons__tertiary.hex}>
+                    <div id="item2">
+                        {mission === placeholderMission ? (
+                            <PaddingLeft>
+                                <DotProgress size={48} color="primary" />
+                            </PaddingLeft>
+                        ) : (
+                            <Button variant="ghost" fullWidth>
+                                <Typography variant="body_short_bold">{mission.name}</Typography>
+                            </Button>
+                        )}
+                    </div>
+
+                    <Typography id="item3" variant="caption" color={tokens.colors.text.static_icons__tertiary.hex}>
                         {TranslateText('Robot')}: {mission.robot.name}
                     </Typography>
-                    <Typography variant="caption" color={tokens.colors.text.static_icons__tertiary.hex}>
+                    <Typography id="item4" variant="caption" color={tokens.colors.text.static_icons__tertiary.hex}>
                         {TranslateText('Tasks')}: {numberOfTasks}
                     </Typography>
-                    <MissionDurationDisplay mission={mission} />
+                    <div id="item5">
+                        <MissionDurationDisplay mission={mission} />
```

Comment:
Consider only displaying if the screen is large enough?

Reply:
If the robot name is too long it will end up under tasks. If we remove estimated duration for small screens we should have ample space
![image](https://github.com/equinor/flotilla/assets/32652118/c6fbc402-dcaf-4eaf-83c1-7f425193934a)

Reply:
I think Ellipsis can be used here and no need display the whole robot name ?

Reply:
I have removed estimated duration when in the iphone and when it's in the laptop it shows up